tests/integration/targets/firewall_update/tasks/main.yaml

```diff
@@ -0,0 +1,613 @@
+tasks:
```

Comment:
@LBGarber: I don't know how you run your test suite. I ran this manually separately with my own harness. For this reason, whatever I put here might not work out of the box with whatever test suite you use.

Reply:
We generally run the following command to execute the e2e test suite:
```
make TEST_ARGS="-v test_target_name" test
```

Fortunately this looks good for the most part, so it shouldn't be too much trouble to get it compatible.